schema.sql

```diff
@@ -37,4 +37,30 @@ ALTER TABLE animals
 COMMIT;
 SELECT * FROM animals;
 
+/*day-4*/
+CREATE TABLE vets (
+    id INT primary key generated always as identity,
+    name VARCHAR(80) NOT NULL,
+    age INT NOT NULL,
+	date_of_graduation DATE
+);
+
+CREATE TABLE specializations (
+    id INT primary key generated always as identity,
+    vets_id INT NULL REFERENCES vets(id),
+	species_id INT NULL REFERENCES species(id)
+);
+
+CREATE TABLE visits (
+    id INT primary key generated always as identity,
+    vets_id INT NULL REFERENCES vets(id),
+	animals_id INT NULL REFERENCES animals(id),
+	visit_date DATE NULL
+);
+
+/* Block-2 */
+/*Day-1*/
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
 
```

Comment:
For `SELECT COUNT(*) FROM visits where animals_id = 4;` and `SELECT * FROM visits where vet_id = 2` queries:

The execution time should be significantly lower than 1 sec (1000 ms). Kindly add the indexes again and re-run the db to see if the performance of queries increases.

Reply:
Hi,

Thank you for the kind review,

During inserting data into tables, we have inserted a huge amount of data, and as you can see CREATE INDEX decreases the time all the way from 6.5 seconds to 1.8 seconds.
And CREATE INDEX cannot be done twice on the same table and column.